app/_src/gateway/upgrade/dual-cluster.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Dual-cluster upgrade
+content_type: how-to
+purpose: Learn how to perform a dual-cluster upgrade for Kong Gateway
+---
+
+The dual-cluster upgrade strategy is a {{site.base_gateway}} upgrade option used primarily for traditional 
+mode deployments and for control planes in hybrid mode. 
+
+With a dual-cluster upgrade, you deploy a new cluster of version Y alongside the current version X, 
+so that two clusters serve requests concurrently during the upgrade process. 
+As you add nodes, you will gradually adjust the traffic ratio between the two clusters to 
+switch traffic over from the old cluster to the new one.
+
+![Dual-cluster upgrade workflow](/assets/images/products/gateway/upgrade/dual-cluster-upgrade.png)
+> _Figure 1: Dual-cluster upgrade workflow_
+
+This method has limitations on automatically generated runtime metrics that rely on the database. 
+For example, if the Rate-Limiting-Advanced (RLA) plugin is configured to store request counters in 
+the database, the counters between database X and database Y are not synchronized. 
+The impact scope depends on the `window_size` parameter of the plugin. 
```

Comment:
Another example:

Similarly, the same limitation applies to [Vitals](https://docs.konghq.com/gateway/latest/kong-enterprise/analytics/) if you have a large amount of buffered metrics in Postgres or Cassandra.

Reply:
This is in here, it's just version-gated. But in hindsight, since this is an upgrade doc, I think it's fair to leave the example unversioned, since they could be coming from any version, where Vitals and Cassandra are both still supported.